scripts/main.js

```diff
@@ -1,20 +1,18 @@
-var myImage = document.querySelector('img');
+<script src="JavaScript/jquery-1.10.2.js" type="text/javascript"></script> 
 
-myImage.onclick = function() {
-    var mySrc = myImage.getAttribute('src');
-    if(mySrc === 'images/Hawaii/echeveriaweb.jpg') {
-      myImage.setAttribute ('src','images/Hawaii/IMG_2486web.jpg');
-    }
-    if(mySrc === 'images/Hawaii/IMG_2486web.jpg') {
-      myImage.setAttribute ('src','images/Hawaii/IMG_2441web.jpg');
-    }
-    if(mySrc === 'images/Hawaii/IMG_2441web.jpg') {
-      myImage.setAttribute ('src','images/Hawaii/IMG_2446web.jpg');
-    }
-    if(mySrc === 'images/Hawaii/IMG_2446web.jpg') {
-      myImage.setAttribute ('src','images/Hawaii/IMG_2462web.jpg');
-    }
-    if(mySrc === 'images/Hawaii/IMG_2462web.jpg') {
-      myImage.setAttribute ('src','images/Hawaii/echeveriaweb.jpg');
-    }
-}
+<script type="text/javascript">
```

Comment:
inside the javascript file you do not need these <script type ...> stuff. Just pure javascript in here.

also remove the jquery import above. it is automagically loaded as long as on the same html page it is imported.